### PR TITLE
Compress tests and ng2 template bindings

### PR DESCRIPTION
**Pull request by jiminys**
Hey, as discussed with extra tests!

I've made a couple of changes to the regex's to remove more whitespace and checked for zero length/null matches. All look/test ok for me, but windows....

### CODE REVIEW

**Comment by pablodenadai** [comment]
Cool, so no luck with Windows? Is it a CRLF related issue?

**Comment by jiminys** [comment]
Windows seems fine, but I can't confirm on anything else (other than travis etc)

**Comment by pablodenadai** [comment]
I see, I'll give it a go on Mac later on. Thanks!

**Comment by jiminys** [comment]
I've yet to add the cli tests, just working on them now.

File: index.js [hunk 1]
```diff
@@ -63,9 +63,14 @@ function processTemplateUrl(content, options) {
 
 		let file = fs.readFileSync(path.join(options.base, url), 'utf-8');
 		if (options.compress) {
-			file = minify(file, {collapseWhitespace: true, removeComments: true});
+			file = minify(file,
+				{
+					collapseWhitespace: true,
+					removeComments: true,
+					ignoreCustomFragments: [/\s\[.*\]=\"[^\"]*\"/, /\s\([^)"]+\)=\"[^\"]*\"/]
```
Comment (pablodenadai) [inline index.js:82]: Is it alright if you add an inline comment/documentation here so we know what the RegEx is doing?

Comment (jiminys) [inline index.js:82]: sure, will do

**Comment by jiminys** [comment]
I think that's me for now. The watch runs fine on our test project, which admittedly isn't that complicated. See what you think. Cheers

**Comment by pablodenadai** [comment]
Good stuff mate. I'll review it later on tonight. Cheers.

Ps. Are you using this lib at work?

**Comment by jiminys** [comment]
yup hence the two jim stamps committing!!

**Comment by pablodenadai** [comment]
Merged. I'll make a few tweaks (https://github.com/ghpabs/angular2-inline-template-style/commit/26bb90a64a0857518c935ba5ffb7d819f2415281) and publish it to npm.

Thanks again.

**Comment by jiminys** [comment]
Np

**Pull request merged**